tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +937,16 @@
     <value>X64 process running on X64 machine</value>
     <comment>Message text when the process is x64 CPU architecture running on an X64 CPU machine.</comment>
   </data>
+  <data name="AliasDisabledDialogButtonText" xml:space="preserve">
+    <value>Go to Advanced app settings</value>
+    <comment>Content Dialog button text which on click opens the Advanced App settings page in Windows Settings app.</comment>
+  </data>
+  <data name="AliasDisabledDialogContent" xml:space="preserve">
+    <value>Go to Advanced app settings -&gt; App execution alias -&gt; Project Ironsides -&gt; Enable the toggle and try to launch PI as admin again</value>

```

Comment:
This string feels weird to me. Is there an established pattern for describing a flow like this in a public UI? (I know we write it this way for ourselves all the time) Can we at least change it to just ">" to match the Breadcrumb UI?

Reply:
updated to just ">". I am open to further suggestions. 
I was hoping the content writers will help improve the text eventually.